### ISSUE TITLE
unmet peer dependency "stimulus@>=3"

### ISSUE THREAD

**Issue by spaquet** [stimulus-use/stimulus-use#172]
Here is my package.json

"dependencies": {
    "@hotwired/stimulus": "3.0.1",
...

but installing stimulus-use is returning the following error message:
`warning " > stimulus-use@0.50.0-2" has unmet peer dependency "stimulus@>=3".`

I also try by adding stimulus-use using `yarn add stimulus-use@beta` but that does not change anything.

Purging the node_modules folder does not change anything either.

**Comment by adrienpoly** [comment]
I added this to try to limit compatibility issues with older packages that would still link to `stimulus` package. 
This being said I agree that there is no better time to fully upgrade to the new package name. 
I will release shortly a version without it. This being said while this warning is annoying it should really hurt neither

**Comment by gremo** [comment]
Any plan to merge Stimulus 3 support and release a new version? Thanks so much for this library!

**Comment by aaronbrethorst** [comment]
Any update?

**Comment by gremo** [comment]
Any update on this? I really link Stimulus 3 new features, but still want to use this awesome library!

**Comment by marcoroth** [comment]
Hey @gremo, we are planning to get full support for Stimulus 3 out soon! See #170 for the updates on that front.

But in the meantime, there is nothing which would break/stop you from using stimulus-use with Stimulus 3 today.

**Comment by gremo** [comment]
Thanks @marcoroth for the quick reply. You mean that I can upgrade to Stimulus 3 and use the latest tag of stimulus-use (stable, not beta)?

**Comment by marcoroth** [comment]
@gremo Exactly, it works with the latest stable version. I know it's not ideal, since you need both `@hotwired/stimulus` and `stimulus`, but it work without any issues.

I have it running in a few apps of mine in production with:
```json
    "@hotwired/stimulus": "^3.0.1",
    "stimulus": "^3.0.1",
    "stimulus-use": "^0.41.0"
 ```

**Comment by gremo** [comment]
Thanks @marcoroth , it works even without it because stimulus is only a _peer dependency_:

        "stimulus": "^3.0.1",
        "stimulus-password-visibility": "^2.0.0",
        "stimulus-use": "^0.41.0",

I was just worried about the actual code, but it seems to work just fine. In fact, the upgrade from Stimulus 2 to 3 is super-easy, a very few BC.

**Comment by marcoroth** [comment]
Good news! We just released [`0.50.0`](https://github.com/stimulus-use/stimulus-use/releases/tag/v0.50.0) which ships native support for Stimulus 3, thanks your for patience!

You should also be able to remove the `stimulus` npm package from your `package.json`, given no other package depends on it.

Please let us know if you see anything unexpected, thank you!

**Comment by aaronbrethorst** [comment]
thanks @marcoroth!
